gradle.properties

```diff
@@ -21,7 +21,7 @@ stdlibLogVersion=2.2.1
 stdlibMimeVersion=2.2.1
 stdlibCryptoVersion=2.2.1
 stdlibAuthVersion=2.2.1
-stdlibHttpVersion=2.2.1
+stdlibHttpVersion=2.3.0-20220428-114700-fce5b3e
```

Comment:
Why do we have to bump only the HTTP version to a timestamped version, but not the other dependencies? Am I missing something in here? :)